nixos/tests/openssh.nix

```diff
@@ -108,6 +108,23 @@ in {
         };
       };
 
+    server_nopam =
```

Comment:
*Very* small nitpick: Since all but `server_allowedusers` use dashes, I'd probably rename `server_allowedusers` and use dashes so that it's consistent.